docs/output.md

```diff
@@ -41,8 +42,8 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
       - [Sentieon DNAscope joint germline variant calling](#sentieon-dnascope-joint-germline-variant-calling)
     - [Sentieon Haplotyper](#sentieon-haplotyper)
       - [Sentieon Haplotyper joint germline variant calling](#sentieon-haplotyper-joint-germline-variant-calling)
-    - [Lofreq](#lofreq)
     - [Strelka](#strelka)
+    - [Lofreq](#lofreq)
```

Comment:
why changing the order of this?

Reply:
In fact I didn't changed this line, this have been done automatically.
But it does reflect that the Strelka paragraph is before Lofreq.

Reply:
I think it's a markdown linting thing